go/pkg/db/key_data.go

```
@@ -0,0 +1,98 @@
+	case *FindLiveKeyByHashRow:
+		kd = buildKeyData(row)
+	case *FindLiveKeyByIDRow:
+		// Cast is safe because structs have identical fields
+		kd = buildKeyData((*FindLiveKeyByHashRow)(unsafe.Pointer(row)))
+	}
+
+	return kd
+}
+
+func buildKeyData(r *FindLiveKeyByHashRow) *KeyData {
+	kd := &KeyData{
```

golangci-lint (2.2.2)

db.KeyData is missing fields Identity, Roles, Permissions, RolePermissions, Ratelimits

(exhaustruct)